.github/workflows/lint-and-test.yml

```diff
@@ -19,15 +19,21 @@ jobs:
         os: [ubuntu-18.04]
         epubcheck: [4.2.4]
         prince: [12.5-1]
-        wordpress: [5.8.1, latest]
+        wordpress: [5.8.2]
         experimental: [false]
         include:
           - php: 7.4
-            os: ubuntu-18.04
+            os: ubuntu-20.04
             wordpress: latest
             epubcheck: 4.2.6
             prince: 14.2-1
             experimental: true
+          - php: 7.4
+            os: ubuntu-20.04
+            wordpress: 'trunk'
```

Comment:
add new tests to test against 'trunk' of WordPress (this will allow us to run tests with PHPUnit 9 before WP 5.9 is released)